docs/source/getting-started/installation.rst

```diff
@@ -660,6 +663,12 @@ Seeweb
     [DEFAULT]
     api_key = <your-api-token>
 
+4. install Seeweb Library:
+
+.. code-block:: bash
+
+   pip install ecsapi
+
```

Comment:
This should be included with `pip install skypilot[seeweb]` above, so no need to mention it here : )